pythonFiles/interpreterInfo.py

```diff
@@ -9,5 +9,6 @@
 obj["sysPrefix"] = sys.prefix
 obj["version"] = sys.version
 obj["exe"] = sys.executable
+obj["is64Bit"] = sys.maxsize > 2 ** 32
```

Comment:
Interesting. I knew about plarform.architecture(), but seems like this is the recommended way.

Reply:
This was always part of the core code, at some point it was removed and looks like it's getting added back again.